pkg/services/accesscontrol/acimpl/service.go

```diff
@@ -790,3 +792,11 @@ func (s *Service) GetRoleByName(ctx context.Context, orgID int64, roleName strin
 	})
 	return role, err
 }
+
+func (s *Service) Check(ctx context.Context, in *openfgav1.CheckRequest) (*openfgav1.CheckResponse, error) {
+	return s.zclient.Check(ctx, in)
+}
+
+func (s *Service) ListObjects(ctx context.Context, in *openfgav1.ListObjectsRequest) (*openfgav1.ListObjectsResponse, error) {
+	return s.zclient.ListObjects(ctx, in)
+}
```

Comment:
Any reason we added this to the service and not `AccessControl` component?

Reply:
That's a good question. I'll try to look if it fits better for AccessControl.

Reply:
I would imagine so, all access related checks already goes through there right now